README.md

```diff
@@ -44,6 +44,14 @@ Spyder may also be used as a PyQt4/PySide extension library (module
 `spyderlib`). For example, the Python interactive shell widget used in
 Spyder may be embedded in your own PyQt4/PySide application.
 
+## Running from source
```

Comment:
Who are we? As I already said, the README.md is for GitHub users, who prefer to work with sources if possible. For everybody else there are other other channels.

Cloning and running is that **fastest** way a person from GitHub can run Spyder. I don't see why we want to put users through the installation rituals.